modules/acquia_cms_common/tests/src/ExistingSite/ContentTypeListTestBase.php

```diff
@@ -208,79 +207,86 @@ public function testListPage(array $permissions = NULL) {
-
-    // Assert all type facets are available.
-    $assert_session->linkExists('Type A (2)');
-    $assert_session->linkExists('Type B (2)');
-    $assert_session->linkExists('Type O (2)');
-
-    // All content should be visible except for the secret one.
-    $this->assertLinksExistInOrder();
-    $assert_session->linkNotExists('Secret');
-    $assert_session->linkNotExists('Spanish - Alpha');
-
-    // Filter by a category and ensure that the expected content is visible.
-    $page = $this->getSession()->getPage();
-    $page->clickLink('Art (2)');
-    // Assert that the clear filter is present.
-    $assert_session->linkExists('Clear filter(s)');
-
-    $assert_session->addressMatches('/.\/category\/art-[0-9]/');
-    $this->assertLinksExistInOrder(['Foxtrot', 'Beta']);
-    $assert_session->linkNotExists('Alpha');
-    $assert_session->linkNotExists('Charlie');
-    $assert_session->linkNotExists('Delta');
-    $assert_session->linkNotExists('Echo');
-    $assert_session->linkNotExists('Secret');
-
-    // The choice of a category should narrow down the results in the type
-    // facet.
-    $assert_session->linkNotExists('Type A');
-    $assert_session->linkExists('Type B (1)');
-    $assert_session->linkExists('Type O (1)');
-
-    // Filtering by type should narrow the results down even more.
-    $page->clickLink('Type O (1)');
-    $assert_session->addressMatches('/.\/type\/type-o-.*\/category\/art-.*/');
-    $assert_session->linkNotExists('Alpha');
-    $assert_session->linkNotExists('Beta');
-    $assert_session->linkNotExists('Charlie');
-    $assert_session->linkNotExists('Delta');
-    $assert_session->linkNotExists('Echo');
-    $assert_session->linkExists('Foxtrot');
-    $assert_session->linkNotExists('Secret');
-
-    // Removing a facet should widen the results.
-    $page->clickLink('Art (1)');
-    $assert_session->addressMatches('/.\/type\/type-o-.*/');
-    $this->assertLinksExistInOrder(['Foxtrot', 'Charlie']);
-    $assert_session->linkNotExists('Alpha');
-    $assert_session->linkNotExists('Beta');
-    $assert_session->linkNotExists('Delta');
-    $assert_session->linkNotExists('Echo');
-    $assert_session->linkNotExists('Secret');
-
-    // Assert translated items on translated-list page.
-    $this->visitListPage('es');
-    $assert_session->linkExists('Spanish - Alpha');
-
-    // Assert filtered nodes on search page.
-    $options = [
-      'query' => ['keywords' => 'Alpha'],
-    ];
-    $this->drupalGet('/search', $options);
-    $assert_session->linkExists('Alpha');
-    $assert_session->linkNotExists('Spanish - Alpha');
-
-    $this->drupalGet('/es/search', $options);
-    $assert_session->linkExists('Spanish - Alpha');
+    // Since below tests would require certain module to be present
+    // in order to pass all test, we are adding check instead of
+    // adding the required module as dependencies for excellent
+    // reasons so that we can run the minimum tests in isolation.
```

Comment:
I'd remove "for excellent reasons" here.

Reply:
Done.